xcube/core/update.py

```diff
@@ -20,19 +20,23 @@
 # SOFTWARE.
 
 import datetime
-from typing import Any, Dict
+from typing import Any
+from typing import Dict
 
 import xarray as xr
 
 from xcube.constants import FORMAT_NAME_NETCDF4
 from xcube.constants import FORMAT_NAME_ZARR
+from xcube.core.gridmapping import GridMapping
 from xcube.util.config import NameDictPairList
 
 _LON_ATTRS_DATA = ('lon', 'lon_bnds', 'degrees_east',
-                   ('geospatial_lon_min', 'geospatial_lon_max', 'geospatial_lon_units', 'geospatial_lon_resolution'),
+                   ('geospatial_lon_min', 'geospatial_lon_max',
+                    'geospatial_lon_units', 'geospatial_lon_resolution'),
                    float)
 _LAT_ATTRS_DATA = ('lat', 'lat_bnds', 'degrees_north',
-                   ('geospatial_lat_min', 'geospatial_lat_max', 'geospatial_lat_units', 'geospatial_lat_resolution'),
+                   ('geospatial_lat_min', 'geospatial_lat_max',
+                    'geospatial_lat_units', 'geospatial_lat_resolution'),
                    float)
 _TIME_ATTRS_DATA = ('time', 'time_bnds', None,
```

Comment:
Should be cleaned up (unused `None` values) and/or defined in the function where it is used. May also remove entirely.